bot/helper/drive_utils/gdriveTools.py

```diff
@@ -123,6 +123,10 @@ def edit_telegraph(self):
         return
 
     def drive_list(self, fileName):
+        if len(fileName) > 2:
+            remove_list = ['A', 'a', 'X', 'x']
```

Comment:
Why not also include the extended stop words list from say....
https://www.ranks.nl/stopwords

What do you think about this?

Reply:
Yeah, we can include the stop words, but I was considering few things.

There are few files such as : 

"Q 2016 1080p"

Now, although "Q" is not in the stopword list, but lets assume it is. 
Then the code will remove Q and the remainin name will become "2016 1080p" and this will match with 100s of files. Even if we do send top 95 resuts, there is a possibility that we won't be able to find the above mantioned file. 

But yeah, number of such files are very low. So we can add stopwords, but just wanted to tell you what issues we might face. If you think we are good to go, then i can add the respective changes.

Reply:
Let's try with current and see how it goes, then push more words as it makes sense.